churchslavonic.tex

```diff
@@ -824,45 +827,34 @@ \subsection{Набор Знаменной нотации}
-\end{EN}
-%
-\begin{RU}
-\cuKinovar{Ограничения}: шрифты для Знаменной нотации обычно предоставляют информацию о цветах некоторых глифов (например, киноварных помет) в таблицах COLR и CPAL. Однако технология COLR и CPAL пока не поддерживается в \XeTeX{} и \LuaTeX{}. В качестве решения проблемы раскраски глифов, Вы можете раскрасить киноварные пометы вручную используя команду \cs{cuKinovar}:
-\end{RU}
+\cuKinovar{Note 3}: fonts for Znamenny Notation usually provide color information for certain glyphs (such as the cinnabar marks) via data in the COLR and CPAL tables. However, COLR / CPAL technology is presently not supported in \XeTeX{} or \LuaTeX{}. The \cs{cuKruk} and \cs{cuKrukPara} commands will automatically typeset the cinnabar marks in red by internally invoking the \cs{cuKinovar} macro. To render the cinnabar marks in grayscale or black, load the \pkg{churchslavonic} package with the \texttt{gray} or \texttt{bw} options, respectively. 
 
-\begin{tabular}{cc}
-\verb+\cuKruk{+{\musicFont } \verb+\cuKinovar{+ {\musicFont } \verb+}}{+ \textchurchslavonic{жда} \verb+}+ &  \textchurchslavonic{\cuKruk{\cuKinovar{}}{жда}} \\
-\end{tabular}
-
-\smallskip
-
-\begin{EN}
-However, this approach breaks normal OpenType positioning rules for the marks in \XeTeX{} (though it does appear to work correctly in \LuaTeX{} using the \pkg{luacolor} package).
-
-\cs{cuKruk} commands may be nested, which can be used to produce `explanatory marks' (\textrussian{<<подобные пометы>>}):
+\cuKinovar{Limitations}: The use of color breaks normal OpenType positioning rules for marks in \XeTeX{}. You can obtain correct positioning of marks in \LuaTeX{} by loading the \pkg{luacolor} package in the preamble of your document.
 
+The \cs{cuKruk} commands may be nested, which can be used to produce `explanatory marks' (\textrussian{<<подобные пометы>>}):
 \end{EN}
 %
 \begin{RU}
-Однако данный подход приведет к неправильному позиционированию раскрашенных глифов в \XeTeX{} (похоже, правильных результатов можно достичь используя \LuaTeX{} и пакет \pkg{luacolor}).
+\cuKinovar{Примечание 3}: шрифты для Знаменной нотации обычно предоставляют информацию о цветах некоторых глифов (например, киноварных помет) в таблицах COLR и CPAL. Однако технология COLR и CPAL на данный момент не поддерживается в \XeTeX{} и \LuaTeX{}. Команды \cs{cuKruk} и \cs{cuKrukPara} автоматически раскрашивают киноварные пометы, внутренне вызывая для этого команду \cs{cuKinovar}. Чтобы набрать киноварные пометы в оттенках серого или черным цветом, следует загрузить пакет \pkg{churchslavonic} с опциями \texttt{gray} или \texttt{bw}, соответственно.
+
+\cuKinovar{Ограничения}: автоматическая раскраска киноварных помет  в \XeTeX{} нарушает работу правил позиционирования глифов в таблицах OpenType. Правильного позицинирования раскрашенных помет можно достичь в \LuaTeX{}, загрузив в преамбуле документа пакет \pkg{luacolor}.
```

Comment:
no need to load `luacolor` as it is being automatically loaded by `cu-kinovar` when LuaTeX is detected...

Reply:
Fixed.